e2e/dashboards-suite/dashboard-templating.spec.ts

```diff
@@ -3,6 +3,13 @@ import { e2e } from '../utils';
 describe('Dashboard templating', () => {
   beforeEach(() => {
     e2e.flows.login(Cypress.env('USERNAME'), Cypress.env('PASSWORD'));
+    // Note: Only works in Chrome/Chromium-based browsers
+    Cypress.automation('remote:debugger:protocol', {
+      command: 'Emulation.setTimezoneOverride',
+      params: {
+        timezoneId: 'America/New_York', // OR  'UTC'
```

Comment:
not sure if relevant, but we use `'Pacific/Easter'` in Jest tests:

https://github.com/grafana/grafana/blob/ec35e861e09f5eab7bfa2431d1a03ff85cd6701e/jest.config.js#L4

Reply:
shouldn't matter since this only impacts these tests.  but i changed it to be consistent